spec/upgrade_helpers.lua

```diff
@@ -179,6 +179,27 @@ local function all_phases(phrase, f)
   return it_when("all_phases", phrase, f)
 end
 
+local get_busted_handler
```

Comment:
Please add a comment that describes the purpose of the function and that mentions that it is used (only) for migration tests.